solr/solrj/src/java/org/apache/solr/client/solrj/impl/ConcurrentUpdateHttp2SolrClient.java

```diff
@@ -64,8 +64,8 @@ public class ConcurrentUpdateHttp2SolrClient extends SolrClient {
 
   private boolean shutdownClient;
   private boolean shutdownExecutor;
-  private int pollQueueTime = 250;
```

Comment:
does this default value need to move to the builder? I'm not sure I am seeing it anywhere

Reply:
Humm....    Good darn question....

Reply:
@stillalex I think that the method  `public Builder setPollQueueTime(long pollQueueTime, TimeUnit unit)`  sets `pollQueueTimeMillis` on the builder, which is then set as `this.pollQueueTimeMillis = builder.pollQueueTimeMillis;`

Reply:
sure. but I was asking about the default value. is that same as before? I'm not seeing it moved over.

Reply:
IntelliJ told me the default is not actually ever used, it's always set..  so I removed it...   However, maybe I made a mistake/  Thanks for looking!